app-text/nfoview/nfoview-1.27.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 gnome2-utils xdg-utils
```

Comment:
gnome2-utils isnt' needed here anymore, and using it for icon_cache_update on an EAPI-7 ebuild shouldnt even work. Use `xdg_icon_cache_update` instead. R

Reply:
ok, good to know. Is changed now.